### PR TITLE
Feature/eval chips

### PR DESCRIPTION
**Pull request by LNdr28**
Changes for creating chips with masked background and training and testing the chip model.

### CODE REVIEW

File: tools/train/eval_chips.py [hunk 1]
```diff
@@ -64,29 +79,39 @@ def train_chips(args):
         f'No log path is given, the default log path will be used: {log_path}')
     logger.info('Initiating the training of the BBAVector model...')
 
-    transform = [torchvision.transforms.RandomHorizontalFlip(), torchvision.transforms.RandomRotation(180),
-                 torchvision.transforms.RandomVerticalFlip()]
+    transform = [torchvision.transforms.RandomVerticalFlip()]
```
Comment (Iammuratc) [inline tools/train/eval_chips.py:82]: isn't this supposed to be horizontal flip?

**Review by Iammuratc — COMMENTED** [review]
Great.

File: tools/train/eval_chips.py (hunk 1 above)
Comment (LNdr28) [reply tools/train/eval_chips.py:82]: Good catch, you're right. The performance increase I've seen when changing that probably stems from the removed random rotation, so the model only had to deal with the incorrect flip. I'll fix that and rerun the experiments, maybe we'll see better results with that.

Comment (Iammuratc) [reply tools/train/eval_chips.py:82]: ok. Let me prepare the sets. I will also try to create the chips for you.

**Pull request merged**